Build/Build.cs

```diff
@@ -275,19 +273,19 @@ public override void ConfigureTestMatrix (TestMatricesBuilder builder)
         "DatabaseTestMatrix",
         new TestDimension[,]
         {
-            { Docker_Win_NET48, NET48, NoBrowser, SqlServer2016, Debug, x86 },
-            { Docker_Win_NET48, NET48, NoBrowser, SqlServer2016, Release, x86 },
-            { Docker_Win_NET48, NET48, NoBrowser, SqlServer2016, Debug, x64 },
-            { Docker_Win_NET48, NET48, NoBrowser, SqlServer2016, Release, x64 },
+            { Docker_Win_NET8_0, NET8_0_WINDOWS, NoBrowser, SqlServer2016, Debug, x86 },
+            { Docker_Win_NET8_0, NET8_0_WINDOWS, NoBrowser, SqlServer2016, Release, x86 },
+            { Docker_Win_NET8_0, NET8_0_WINDOWS, NoBrowser, SqlServer2016, Debug, x64 },
+            { Docker_Win_NET8_0, NET8_0_WINDOWS, NoBrowser, SqlServer2016, Release, x64 },
 
             // Local-->
-            { LocalMachine, NET48, NoBrowser, SqlServerDefault, Debug, x86 },
+            { LocalMachine, NET8_0_WINDOWS, NoBrowser, SqlServerDefault, Debug, x86 },
 
             // Exercise compatibility between installed .NET version, target framework and SQL Server
-            { Docker_Win_NET481, NET481, NoBrowser, SqlServer2022, Release, x64 },
-            { Docker_Win_NET481, NET48, NoBrowser, SqlServer2019, Release, x64 },
-            { Docker_Win_NET48, NET48, NoBrowser, SqlServer2017, Release, x64 },
-            // { Docker_Win_NET48 , NET48, NoBrowser, SqlServer2016, Release, x64 },
+            { Docker_Win_NET8_0, NET8_0_WINDOWS, NoBrowser, SqlServer2022, Release, x64 },
+            { Docker_Win_NET8_0, NET8_0_WINDOWS, NoBrowser, SqlServer2019, Release, x64 },
+            { Docker_Win_NET8_0, NET8_0_WINDOWS, NoBrowser, SqlServer2017, Release, x64 },
+            // { Docker_Win_NET8_0 , NET8_0_WINDOWS, NoBrowser, SqlServer2016, Release, x64 },
             // { Docker_Win_NET50 , NET50, NoBrowser, SqlServer2022, Release, x64 },
             // { Docker_Win_NET50 , NET50, NoBrowser, SqlServer2019, Release, x64 },
             // { Docker_Win_NET50 , NET50, NoBrowser, SqlServer2017, Release, x64 },
```

Comment:
The docker image only support x64 and will fail for any x86 configuration. We should change this to use x64 instead.